### DISCUSSION TITLE
50% width is not a design token

### DISCUSSION THREAD

**Discussion by krnan9525** [twilio-labs/paste#753] [General] [answered]
I want to use:
```<Flex width={'50%'} hAlignContent={'right'}>..............</Flex>``` 

It warns me 50% is not a design token.
But it doesn't exist in https://paste.twilio.design/tokens/ anyways.
What's the recommended approach without adding unnecessary elements? 

Thanks

**Comment by SiTaggart** [comment, marked as answer]
Yeah, this is a known limitation right now. We're going to be allowing arbitrary height and width values in the near future, I'm just not entirely sure when it'll land.

Curious if you can share a screenshot of the layout you're trying to build, there maybe a way around it with another component or set of components.
